modules/support/client/components/BlockMemberLink.component.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * A link to beport member
+ * @param {string} username - the username of the user to report
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import '@/config/client/i18n';
+import { useTranslation } from 'react-i18next';
+
+import * as api from '../api/block.api';
+
+
+
+
+export default function BlockMemberLink({ username }) {
+  const { t } = useTranslation('support');
+
+  async function handleBlockMember({ username }) {
```

Comment:
The `{}` brackets here make JS think this the function parameter should be an object, with key `username`:

```js
{ username:  'something' }
```

so changing it to ` handleBlockMember(username)` will help.